testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/InProcessGradleExecuter.java

```diff
@@ -280,67 +262,6 @@ protected GradleHandle createGradleHandle() {
         return super.createGradleHandle();
     }
 
-    @Override
-    protected Factory<BaseExecHandleBuilder> getExecHandleFactory() {
```

Comment:
The embedded test executor had special logic for launching forked Gradle processes in case it was forced to execute in forking mode. 

However embedded integ tests must have a valid distribution, so we can instead fallback to the existing logic that the forking executor uses for launch Gradle processes